### PR TITLE
Gaprindashvili Beta2 Announce

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@cybette all smoke testing has passed. Please review and merge is all looks good.

File: site/_posts/2017-11-30-Announcing-Gaprindashvili-Beta2.html.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Announcing Gaprindashvili Beta Release
```
Comment (cybette) [inline site/_posts/2017-11-30-Announcing-Gaprindashvili-Beta2.html.md]: should be "Beta 2 Release"

File: site/_posts/2017-11-30-Announcing-Gaprindashvili-Beta2.html.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Gaprindashvili is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. Here are just a few of the new or updated features.
+
+* Alerts - Seed MiqAlerts used for Prometheus Alerts [(#16479)](https://github.com/ManageIQ/manageiq/pull/16479)
+* Ansible - Add log_output option for embedded ansible service [(#16414)](https://github.com/ManageIQ/manageiq/pull/16414)
+* Replication - Prevent replication subscription to the same region as the current region [(#16446)](https://github.com/ManageIQ/manageiq/pull/16446)
+* RBAC - Standalone ServiceUI product features require, updating affected roles [(#16329)](https://github.com/ManageIQ/manageiq/pull/16329)
+* Providers - Refresh new target do not run post_refresh [(#16436)](https://github.com/ManageIQ/manageiq/pull/16436)
+* i18n - Add Middleware Server EAP/Wildfly translation [(#16492)](https://github.com/ManageIQ/manageiq/blob/gaprindashvili/CHANGELOG.md/)
```
Comment (cybette) [inline site/_posts/2017-11-30-Announcing-Gaprindashvili-Beta2.html.md]: Wrong link (should link to PR not changelog)

**Comment by cybette** [comment]
One more thing: need to change Beta1 to Beta2 in site/_data/releases.yaml

Thanks @JPrause !

**Pull request merged**